pkl-codegen-kotlin/src/main/kotlin/org/pkl/codegen/kotlin/Main.kt

```diff
@@ -80,6 +81,18 @@ class PklKotlinCodegenCommand :
         help = "Whether to make generated classes implement java.io.Serializable"
       )
       .flag()
+  
+  private val packageMapping: Map<String, String> by
+    option(
+      names = arrayOf("-m", "--package-mapping"),
```

Comment:
See comment earlier about short names

```suggestion
      names = arrayOf("--package-mapping"),
```

Reply:
Fixed!